src/Factory.php

```diff
@@ -57,7 +57,7 @@ public function createServer($address)
         $loop = $this->loop;
 
         return $this->resolveAddress($address)->then(function ($address) use ($loop) {
-            $socket = @stream_socket_server($address, $errno, $errstr, STREAM_SERVER_BIND);
+            $socket = @\stream_socket_server($address, $errno, $errstr, STREAM_SERVER_BIND);
```

Comment:
Changes LGTM, though the same should probably also be applied to global constants, no? Can you amend the PR accordingly? :+1:

Reply:
Yes it should I'll amend the PR on the next train :+1: